server/src/main.ts

```diff
@@ -33,7 +33,7 @@ async function bootstrap() {
     }),
   );
 
-  await app.listen(3000);
+  await app.listen(process.env.PORT || 3000);
```

Comment:
I rather use the `EnvironmentService` here instead, and add `PORT` in the `environment.validation.ts`